source/_patterns/04-components/site-logo/site-logo.md

```diff
@@ -2,8 +2,7 @@
 el: .site-logo
 title: Site Logo
 ---
-_Component for Site Logo_
 
-## Variables
-* url: [string] The site link url.
-* site_logo: [string] The path to logo image.
+__Variables:__
+* url: [string] URL of the site logo.
```

Comment:
This reads to me like the url parameter is the URL where the site logo is located (e.g. the path to the image file), which is what the site_logo variable is used for instead. Maybe "URL of the site homepage" or, since we may not want to prescribe that the logo links to the homepage, "URL to which the site logo links."